lessons/Algebra2/exploring-quadratic-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,214 @@
+- Working in pairs or small groups, complete the rest of of @printable-exercise{pages/exploring-mpg.adoc}.
+
+By now we've tried to fit quite a few linear models to this data, and we're not getting anywhere.
+
+[cols="^.^1a,^.^1a,^.^1a", frame="none", stripes="none"]
+|===
+| We can build linear models that fit the increasing half of the data...
+| We can build linear models that fit the decreasing half of the data...
+| We can build linear models that try to split the difference...
+
+| @image{images/fuel-efficiency-linear1.png}
+| @image{images/fuel-efficiency-linear2.png}
+| @image{images/fuel-efficiency-linear3.png}
+|===
+
+...but all of them are so far away from so many points, and our @math{R^2} is pretty bad.
+
+=== Synthesize
+
+- Why didn't linear models work for us?
+- What kinds of curves did you draw to fit better?
+- Can you think of functions that look like those curves?
+
+== Quadratic Relationships @duration{10 minutes}
+
+=== Overview
+Students are introduced to quadratic relationships in the real world, then learn about them in abstract, mathematical terms. They develop vocabulary for describing parts of a parabola.
+
+=== Launch
+
+Linear models are great for fitting simple relationships. But the real world isn't so simple, and, more often than not, linear models are just too primitive to handle that complexity.
+
+Relationships shaped like this are called _quadratic relationships_. They form shapes that look like arches, called parabolas. Sometimes the data only includes the _rising_ part of the parabola, sometimes it only includes the _falling_ part, and and sometimes it includes both.
+
+**Quadratic relationships show up all the time:**
+
+- When an athlete is young, they improve as they get stronger and more skilled. But as they age, they begin to lose their speed and strength.
+- A ball dropped from a tall building will get faster and faster as it falls.
+- Many nutrients are good for you, and the more you take the healthier you are...up to a point. After that, too much of a mineral or vitamin can cause problems.
+
+[.strategy-box, cols="1", grid="none", stripes="none"]
+|===
+
+|
+@span{.title}{Going Deeper: Interaction Effects}
+
+We often think about variables of a model being _independent_ from one another, each contributing a little to the outcome we are measuring. But sometimes one input will amplify the result of another input, meaning their combined contribution is greater than the sum of their separate contributions.
+
+A quadratic relationship often means that one or more variables is _interacting_ with another variable. In our miles-per-gallon example, it turns out that a linear increase in speed has a _more than linear effect_ on mpg!
+|===
+
+[.lesson-instruction]
+Can you think of other real-world examples where a relationship is likely quadratic?
+
+=== Investigate
+
+You already know that linear relationships can be described with terms like "slope" and "y-intercept". Parabolas have their own terms:
+
```

Comment:
This is a huge improvement... it could be more inquiry based... and the glossary definition of vertex (as is the case with many other materials) references the axis of symmetry. 

- Will parabolas have y-intercepts? Will parabolas have slopes?
- What other parts of the parabola do you think we might want to describe?

Reply:
I love this idea. Done.